test/components/idea_generation_lower_third_test.js

```diff
@@ -125,4 +129,66 @@ describe("IdeaGenerationLowerThird component", () => {
       expect(lowerThird.props().displayContents).to.eql(true)
     })
   })
+
+  context("when the stage is voting", () => {
+    it("does not render IdeaSubmissionForm", () => {
+      const lowerThird = shallow(
+        <IdeaGenerationLowerThird
+          {...defaultProps}
+          stage={votingStage}
+        />
+      )
+
+      expect(lowerThird.find(IdeaSubmissionForm)).to.have.length(0)
+    })
+
+    it("renders 5 Votes Left for a user that hasn't voted yet", () => {
+      const lowerThird = shallow(
+        <IdeaGenerationLowerThird
+          {...defaultProps}
+          stage={votingStage}
+        />
+      )
+
+      const votesLeft = lowerThird.find(".thirteen")
+
+      expect(votesLeft.text()).to.equal("5 Votes Left")
+    })
+
+    it("renders the Votes Left for the currentUser", () => {
+      const userWithFiveVotes = {
+        is_facilitator: false,
+        vote_count: voteMax,
+      }
+      const lowerThird = shallow(
+        <IdeaGenerationLowerThird
+          {...defaultProps}
+          stage={votingStage}
+          currentUser={userWithFiveVotes}
+        />
+      )
+
+      const votesLeft = lowerThird.find(".thirteen")
+
+      expect(votesLeft.text()).to.equal("0 Votes Left")
+    })
+
+    it("renders singular Vote if the user has one vote left", () => {
```

Comment:
Nice thinking here, but it the dynamic pluralization/singularization is a bit distracting in that the width of the text changes here. Let's kill this functionality and just have static text that says "Vote(s) Left"